src/kOS/Binding/FlightControlManager.cs

```diff
@@ -150,6 +156,14 @@ private void AddNewFlightParam(string name, SharedObjects shared)
             flightParameters.Add(name, new FlightCtrlParam(name, shared));
```

Comment:
Would it be better to do this as:
```
flightParemeters[name] = new FlightCtrlParam(name,shared);
```
That way, you don't have to bother having protections checking if the key is already present, because unlike Add(), the index operation will happily clobber a value in-place if it already exists (where Add() throws an exception).

The effect this change would have is that every time you call AddNewFlightParam, you'd be guaranteed that it *WOULD* ensure that the value is updated to the new thing, even if it already existed.  Thus preventing potential stale data.  What this class has to do at the moment (As seen below in AddMissingFlightParam() is this:
```
            if (!flightParameters.ContainsKey(name))
            {
                AddNewFlightParam(name, shared);
            }
```
That has the effect that it will refuse to *update* a stale old value if, say, the ``shared`` object is now something different and needs updating (i.e. like after a dock/undock maybe?)